elastic/kibana
# Save the number of executed/scheduled actions

fixes: #122545

Save the number of executed/scheduled actions of the last rule execution in the rule saved object and the event log.

To test: 
- Create a rule and an (or some) action
- Make it generate an alert.
- See the `numberOfExecutedActions` field in the rule saved object and `number_of_executed_actions` in the event log under `kibana.alert.rule.execution.metrics`
